stable/msoms/templates/omsagent-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "msoms.fullname" . }}
+  labels:
+    app: {{ template "msoms.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+ updateStrategy:
+    type: RollingUpdate
+ template:
+  metadata:
+   labels:
+    release: {{ .Release.Name }}
+    app: {{ template "msoms.name" .   }}
+   annotations:
+    agentVersion: {{ .Values.omsagent.annotations.agentVersion }}
+    dockerProviderVersion: {{ .Values.omsagent.annotations.dockerProviderVersion }}
+  spec:
+   containers:
+     - name: {{ template "msoms.name" . }}
+       image: "{{ .Values.omsagent.image.registry }}{{ .Values.omsagent.image.org }}/oms:{{ .Values.omsagent.image.tag }}"
+       imagePullPolicy: {{ .Values.omsagent.image.pullPolicy }}
+       securityContext:
+         privileged: true
+       ports:
+       - containerPort: {{ .Values.omsagent.containers.ports.containerPort.TCP }}
+         protocol: TCP
+       - containerPort: {{ .Values.omsagent.containers.ports.containerPort.TCP }}
+         protocol: UDP
+       volumeMounts:
+        - mountPath: /var/run/docker.sock
+          name: docker-sock
+        - mountPath: /var/log
+          name: host-log
+        - mountPath: /etc/omsagent-secret
+          name: omsagent-secret
+          readOnly: true
+       livenessProbe:
+        exec:
+         command:
+         - /bin/bash
+         - -c
+         - ps -ef | grep omsagent | grep -v "grep"
+        initialDelaySeconds: 60
+        periodSeconds: 60
+   volumes:
```

Comment:
Can you add some sane default resources?

Reply:
EG. https://github.com/samisms/charts/blob/master/stable/gcloud-sqlproxy/templates/deployment.yaml#L23